org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ide.tests.server
+
+import org.junit.Test
+
+import static org.junit.Assert.*
+
+class UriExtensionsTest extends AbstractTestLangLanguageServerTest {
```

Comment:
It would be to test different  `URI` <-> `string` conversions with an instance of `UriExtension`.

```xtend
@RunWith(XtextRunner)
@InjectWith(TestLanguageIdeInjectorProvider)
class UriExtensionsTest extends Assert {

	@Inject
	extension UriExtensions;
	
	@Test
	def void testConversion_01() {
		assertEquals("file:///dir/name.ext", "file:///dir/name.ext".toUri.toPath)
	}
	
	@Test
	def void testConversion_02() {
		assertEquals("file:///dir/name.ext", URI.createFileURI("/dir/name.ext").toPath)
	}

}
```

and so on